src/Model/Export/Product.php

```diff
@@ -637,16 +643,15 @@ public function setProperties(): void
     public function setInputs(): void
     {
         $result = [];
+        $options = $this->item->getOptions() === null
```

Comment:
Not sure what's behind `getOptions()` but assuming it's a DB query? In that case I'd suggest saving the result to a variable instead of calling the function twice.